File: java/Baju.java
public class Baju extends Aksesoris {
    // Additional attributes
    private String untuk;
    private String size;
    private String merk;
    
    // Constructor
    public Baju(int id, String nama_produk, int harga_produk, int stok_produk,
                String jenis, String bahan, String warna,
                String untuk, String size, String merk) {
        // Call the parent constructor
        super(id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna);
        
        // Initialize additional attributes
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }
    
    // Getter methods
    public String getUntuk() {
        return untuk;
    }
    
    public String getSize() {
        return size;
    }
    
    public String getMerk() {
        return merk;
    }
    
    // Setter methods
    public void setUntuk(String untuk) {
        this.untuk = untuk;
    }
    
    public void setSize(String size) {
        this.size = size;
    }
    
    public void setMerk(String merk) {
        this.merk = merk;
    }
}

File: python/Baju.py
from Aksesoris import Aksesoris

class Baju(Aksesoris):
    # Constructor
    def __init__(self, id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna, untuk, size, merk):
        # Memanggil constructor dari parent class (Aksesoris)
        super().__init__(id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna)
        
        # Atribut tambahan untuk Baju
        self.__untuk = untuk
        self.__size = size
        self.__merk = merk
    
    # Getter methods
    def get_untuk(self):
        return self.__untuk
    
    def get_size(self):
        return self.__size
    
    def get_merk(self):
        return self.__merk
    
    # Setter methods
    def set_untuk(self, untuk):
        self.__untuk = untuk
    
    def set_size(self, size):
        self.__size = size
    
    def set_merk(self, merk):
        self.__merk = merk